### DISCUSSION TITLE
[Question][Hw3] Weird syntax error

### DISCUSSION THREAD

**Discussion by mrpositron** [prosyslab-classroom/cs348-information-security#533] [Q&A]
When I run `make` command I got `Syntax error` that says that parenthesis are unmatched. However, I matched them properly and I do not understand what is causing this error. Could you check my latest commit in instrument.ml?

**Comment by KihongHeo** [comment]
Hi.

TAs will help for problems such as undocumented LLVM API, ambiguous specification, basics of functional programming, etc.
However, I expect you folks to learn at least language syntax by yourself from several materials we gave. I believe it is also a part of learning in CS. You should be able to learn any language quickly for your end goal.

**Comment by mrpositron** [comment]
Okay, got it. Thanks.

**Comment by mrpositron** [comment]
I just find out that I named one variable with `rec`. Since `rec` is used to declare recursive function it raised an error. :)
